### DISCUSSION TITLE
Can JWT return when auth login with API?

### DISCUSSION THREAD

**Discussion by imknight** [appwrite/appwrite#4577] [Q&A] [answered]
Trying to explore integrate with Appwrite with just API, for the authentication part it seems like the JWT token can only get it after you login, how to get the JWT token and keep it for next request?

**Comment by stnguyen90** [comment, marked as answer]
Appwrite's session is stored in the `x-fallback-cookies` response header and/or `a_session_<PROJECT ID>` cookie. You can try to extract those and reuse them for future requests.

**Reply by imknight** [reply]
I m  using flutterflow and it seems like unable to parse the cookie data, any other suggestions ? thanks

**Reply by stnguyen90** [reply]
Perhaps this will help you: [Acessing response headers](https://docs.flutterflow.io/data-and-backend/api-calls/api-calls-101#accessing-response-headers).

**Reply by imknight** [reply]
still having problem , seem like flutterflow don't support cookie additional header, any way thanks for pointing me to the right direction

**Reply by stnguyen90** [reply]
Grabbing the `x-fallback-cookies` header should work.

**Reply by imknight** [reply]
ya I can grab the header, and I should use it as cookie header for my follow up request right?
the problem is flutterflow seems like remove "cookie" header.

**Reply by stnguyen90** [reply]
For detailed guidance on connecting FlutterFlow to Appwrite, check out [this](https://medium.com/@stnguyen90/connecting-flutterflow-to-appwrite-56abdbcbbd92) article.

**Reply by imknight** [reply]
lol, u write the article that I have the intention to write! anyway thanks for the guide, now I know the header can just use X-Fallback-Cookies directly, I thought it need to convert into "cookie"
